pyproject.toml

```diff
@@ -13,8 +13,8 @@ license = {text = "Apache-2.0"}
 dynamic = ["version"]
 requires-python = ">=3.9"
 dependencies = [
-    "greenlet==3.1.1",
-    "pyee==12.1.1",
+    "pyee>=12.1.1",

```

Comment:
Make sure to relax the version in the `meta.yaml` as well for our conda customers.

Reply:
mmmmh yeah it's not great for this to not be DRY. I'm not familiar enough with conda to know the best practices on that side of the fence. Here's a related GPT thread discussing the different options/pros/cons -> https://chatgpt.com/share/67476ce2-0288-8010-bc05-e20ff72d15c0

Reply:
I don't like the fact conda doesn't seem to be compatible with dependabot. Is it a requirement to support conda? Seems it's fading https://theregister.com/2024/08/08/anaconda_puts_the_squeeze_on/

Reply:
Yes, I'd prefer to not drop it for now. Updating it manually on conda-side is good enough for us.